src/EnergyPlus/WaterCoils.cc

```diff
@@ -2364,6 +2364,11 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
                 TempSize = AutoSize; // get the autosized air volume flow rate for use in other calculations
             }
 
+            if (state.dataWaterCoils->WaterCoil(CoilNum).WaterCoilModel == iCoilModel::CoolingDetailed) { // Coil:Cooling:Water:DetailedGeometry
+                TempSize = AutoSize; // get the autosized air volume flow rate for use in other calculations
+                state.dataWaterCoils->WaterCoil(CoilNum).DesAirVolFlowRate = AutoSize;
+            }
+
             bool errorsFound = false;
```

Comment:
if state.dataWaterCoils->WaterCoil().DesAirVolFlowRate is zero, which is the case for this coil type, leads to zero value for WaterCoil(CoilNum).InletAirMassFlowRate. As the result other detailed cooling coil sizing parameters such minimum Coil Area and Fin Area will be sized to zero and the sizing calculation fatal out.  So, needed to autosize it but not report it. May be I need to set it to autosize if the value of WaterCoil().DesAirVolFlowRate is zero. I will make changes after reviewing the regression results.